libs/foundry-dev-tools/src/foundry_dev_tools/clients/build2.py

```diff
@@ -48,3 +70,124 @@ def api_get_job_report(self, job_rid: api_types.Rid, **kwargs) -> requests.Respo
             error_handling=ErrorHandlingConfig({204: ResourceNotFoundError}),
             **kwargs,
         )
+
+    def api_submit_build(
```

Comment:
This api method is extremely low level. Can you create a wrapper method for the usual case where someone wants to build a dataset on a branch?

```
submit_build(dataset_rid:str, branch="master", force_build=False)
```

Thanks!